admin/models/restart-code.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+const Schema = mongoose.Schema
+
+const RestartCode = new Schema({
+  status: {
+    type: String,
+    required: true
+  },
+  code: {
+    type: Number,
```

Comment:
I was expecting the restart code to be a short 2 or 3 letter code, rather than a sequence.  I'm happy if @GuyHarwood is happy.

The dev benefit to a short code is that is has more meaning, so makes code understandable.  The integer is effectively meaningless, making on-going development that much harder.

e.g.

```
Code: REM      Description: Remove restart
Code: TKN      Description: Restart taken
Code: MAX     Description: Maximum number of restart already taken
```

I'm sure someone can suggest an improvement here.

Reply:
would rather use codes tbh.  

reasons for not taking check are numbers, but ive made some notes about potentially changing that once we move to SQL.

Reply:
@GeorgeChatzigiannis given that the Ids are numeric, lets go with short alpha codes.  they feel superfluous as numbers.

Reply:
@GuyHarwood @jon-shipley updated